### DISCUSSION TITLE
Where is Shop Page Template?

### DISCUSSION THREAD

**Discussion by choskins-godaddy** [bootscore/bootscore#338] [Q&A] [answered]
When I set the "shop page" in the woocommerce product settings, it changes the template so that it is inside of a container.  I want my main page to be the shop page but I don't want it to be in a container.  So, I'm wondering, where is the code that makes changes based off of that setting.

Thank you so much.

**Comment by choskins-godaddy** [comment]
I figured out the line that I need to change in the shop page.  It's a class that I need to remove in the "content" div.  The class is "container".   This is just on my main page which will be the shop page.

Shop Page:
`<div id="content" class="site-content container py-5 mt-4">`

Regular Page:
`<div id="content" class="site-content">`

**Comment by choskins-godaddy** [comment, marked as answer]
I figured out a solution -- copy the woocommerce.php into my child-theme and remove that class.

**Reply by crftwrk** [reply]
Of course, you can override `woocommerce.php` in child and remove the class `container` there, but this affects **all** shop pages like products, categories etc. 

Why not simply create a new page, choose `Blank` template, add some HTML and a WC shortcode like `[products]` (or any other shortcode provided by WooCommerce https://woocommerce.com/document/woocommerce-shortcodes/) to a Gutenberg Custom HTML block and set this page as the frontpage?

**Reply by choskins-godaddy** [reply]
I played around with short codes. Very cool. I do want to customize the way the products are displayed, however. I want them to be displayed in one row with forward and back arrows placed on top of the left and right cards. I only want this to be displayed on the main page.  Is this the kind of thing I should do in a separate piece of code somewhere?

Also, I wanted to mark your response as the answer so I unmarked mine but it doesn't seem to let me do that. I'm going to mark it answered again.

**Reply by crftwrk** [reply]
>  I want them to be displayed in one row with forward and back arrows placed on top of the left and right cards

https://bootscore.me/documentation/bs-swiper/ Use bS Swiper plugin and modify template as you want.

Same thing here https://github.com/bootscore/bootscore/discussions/226

**Reply by choskins-godaddy** [reply]
Dude!  You are opening up my mind.  I'm so excited but I have to work.  Work now, play later...

**Reply by choskins-godaddy** [reply]
It looks like I can't get products by tag, only by category, is that correct @crftwrk?

That leads me to the question, where is the actual code that implements this?  I see things being passed into arrays but I'm not sure where they are implemented.  The Woocommerce product shortcode supports tags, I just want to compare the implementation of Woocommerce's product shortcode vs Bootscore product shortcode.

It almost feels like Wordpress over abstracts things.  I'm used to seeing code that sends a sql query into the database and gets stuff back.

Thanks so much for your help

**Reply by crftwrk** [reply]
Not with the product template. But you can use any other template to show custom post types by tags. For example the `sc-swiper-card.php` template:

`[bs-swiper-card type="product" tax="product_tag" terms="hats, shirts" order="DESC" orderby="date" posts="5"]`

Copy code from `sc-swiper-card-product.php` inside `<div class="swiper-wrapper">` ... `</div>` and insert it in your `sc-swiper-card.php` to show add to cart button and stars.
